README.md

```diff
@@ -1,137 +1,145 @@
+Swift-Kuery is a pluggable SQL database driver/SDK abstraction layer. Its main idea is to unify the APIs to the various relational databases, providing a Swifty yet SQL-like API. This allows easy switching between databases and forms the basis for an Object-Relational Mapping (ORM) framework.
 
-While Swift-Kuery is not an Object-Relational Mapping (ORM), it provides a great basis to build an ORM. It is also useful if you don't want to commit to a specific database, allowing easy switch between databases.
+[Swift-Kuery-ORM](https://github.com/IBM-Swift/Swift-Kuery-ORM) is an ORM, built on top of Swift-Kuery, which allows you to simplify the persistence of model objects with your server.
 
 Swift-Kuery is easy to learn, consumable framework that comes with a set of [implemented plugins](#list-of-plugins).
 
 ## Table of Contents
-* [Example](#example)
+* [Examples](#examples)
 * [SQL Injection Prevention using Parameterization](#sql-injection-prevention-using-parameterization)
 * [Prepared Statements](#prepared-statements)
 * [Schema Management](#schema-management)
 * [Query Examples](#query-examples)
 * [List of plugins](#list-of-plugins)
 * [License](#license)
 
-## Example
-This example demonstrates how to execute an SQL query using Swift-Kuery and Swift-Kuery-PostgreSQL. It assumes there is a PostgreSQL server running on localhost:5432, that contains a table called Grades:
+## Examples
+This example demonstrates how to execute an SQL query using Swift-Kuery with the [Swift-Kuery-PostgreSQL](https://github.com/IBM-Swift/Swift-Kuery-PostgreSQL) plugin.
 
-```
- id   |  course   | grade
-------+-----------+-------
-12345 | physics   |    82
-12345 | chemistry |    90
-12345 | history   |    98
-78901 | history   |   100
-78901 | chemistry |    94
-78901 | physics   |    90
-24680 | physics   |    74
-24680 | chemistry |    92
-24680 | history   |    90
+The starting point is a Swift project, created by `swift package init --type executable`.
+
+### Creating A PostgreSQL Database
+
+1. Install PostgreSQL
+#### Mac
+`brew install postgresql`
+#### Ubuntu Linux
+`sudo apt-get install postgresql postgresql-contrib`
+
+2. Create a `school` database
+    ```
+    createdb school
+    psql school
+    ```
 
+3. Create a `grades` table
+    ```
+    CREATE TABLE grades (
+        id varchar(100) PRIMARY KEY,
+        course text NOT NULL,
+        grade integer
+    );  
+    ```
+
+### Update your Package.swift file
+Add Swift-Kuery-PostgreSQL to your Package.swift. This will bring in Swift-Kuery as well.
+
+```swift
+dependencies: [
+    ...
+    // Add this line
+    .package(url: "https://github.com/IBM-Swift/Swift-Kuery-PostgreSQL.git", from: "1.0.0"),
+  ],
+  targets: [
+    .target(
+      name: ...
+      // Add this modules to your target(s)
+      dependencies: [..., "SwiftKueryPostgreSQL"]),
+  ]
 ```
 
-First we import Swift-Kuery and Swift-Kuery-PostgreSQL:
+### Send SQL querys to the database
```

Comment:
I don't like the title "Send SQL querys to the database" how about "Executing SQL queries"?